main.go

```diff
@@ -229,12 +235,52 @@ func visit(url *url.URL) {
 				// 30x but no Location to follow, give up.
 				return
 			}
-			log.Fatal("unable to follow redirect: %v", err)
+			log.Fatalf("unable to follow redirect: %v", err)
 		}
 		visit(loc)
 	}
 }
 
+func responseBody(response io.ReadCloser) string {
+
+	var path string
+	var body string
+	var err error
+	if showBodyEnv := os.Getenv(HTTPSTAT_SHOW_BODY); showBodyEnv != "" {
```

Comment:
suggestion: write a GetEnvOrDefault wrapper which lets you always return a valid default that can be parsed. So you can do this
```go
showBody, err := strconv.ParseBool(GetEnvOrDefault("HTTPSTAT_SHOW_BODY", "1"))
```
Looking at this code, does it need to be a boolean? or simply no blank. Ie as a user do I have to do this
```
HTTPSTAT_SHOW_BODY=true ./httpstat ...
```
or is this enough
```
HTTPSTAT_SHOW_BODY=1 ./httpstat
```